app/controllers/User.scala

```diff
@@ -535,6 +535,15 @@ final class User(
       }
     }
 
+  def toggleDoxNote(id: String) =
```

Comment:
it's better to have the client send the desired boolean value, so that the button behavior is predictable

examples:
https://github.com/lichess-org/lila/blame/85654a49a1553b4e0a6e030891f8a1e29413a8f4/conf/routes#L521-L524

Reply:
Makes sense